nodes/src/nodes/branch/IInstance.py

```
@@ -0,0 +1,106 @@
+# SOFTWARE.
+# =============================================================================
+
+from rocketlib import IInstanceBase
+from ai.common.schema import Question, Answer
+from .IGlobal import IGlobal
+
+
+class IInstance(IInstanceBase):
+    IGlobal: IGlobal
+
+    def _build_data(self, text: str, metadata: dict = None, score: float = 0.0) -> dict:
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)